packages/launcher-component/src/pickers/user-repository-picker.tsx

```diff
@@ -31,13 +32,15 @@ const isExistingRepository = (repositories: string[], props: UserRepositoryPicke
 };
 
 export const UserRepositoryPicker: Picker<UserRepositoryPickerProps, UserRepositoryPickerValue> = {
-  checkCompletion: value => (!value.org || REPOSITORY_VALUE_REGEXP.test(value.org)) && REPOSITORY_VALUE_REGEXP.test(value.name || ''),
+  checkCompletion: value => !value.isDuplicate
+    && (!value.org || REPOSITORY_VALUE_REGEXP.test(value.org)) && REPOSITORY_VALUE_REGEXP.test(value.name || ''),
   Element: props => {
     const name = props.value.name || '';
     const helperRepoInvalid = isExistingRepository(props.gitInfo.repositories, props) ?
       `Repository already exists ${valueToPath(props.value, props.gitInfo.login)}` : 'Invalid repository name';
-    const isRepoValid = UserRepositoryPicker.checkCompletion(props.value)
-      && !isExistingRepository(props.gitInfo.repositories, props);
+    const onChange = (value) => {
+      props.onChange({ ...props.value, ...value, isDuplicate: !isExistingRepository(props.gitInfo.repositories, props) });
```

Comment:
I didn't test but: `isDuplicate: !isExisting..` seems weird (the `!`). 

Maybe you should rename the method to `isDuplicateRepository()` to avoid confusion.

Reply:
@ia3andy not only is weird, it wasn't working... fixed it now